test/run-k8s-integration.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+
```

Comment:
Should do an ensure_var on E2E_GOOGLE_APPLICATION_CREDENTIALS and GOPATH?

Also maybe ensure GOPATH is consistent with $BASH_SOURCE[0] to prevent craziness, eg gvm which can set GOPATH in strange ways.

(I know this was copy-paste from the PD driver but that doesn't mean it's right :-))

Reply:
1. With nounset, the script would fatal if E2E_GOOGLE_APPLICATION_CREDENTIALS or GOPATH is unset?
2. Can you please elaborate on the comment? "ensure GOPATH is consistent with $BASH_SOURCE[0]"

Reply:
$BASH_SOURCE[0] is the location of this script. Are we assuming that GOPATH is pointing to this same repository, or could we have some sort of hilarity where GOPATH is pointing to a different version? That can happen, eg, with gvm, or if you are running this out of its own subdirectory but have another version of this in ~/go/src or whatever.

Reply:
Ah, I missed the nounset, thanks

Reply:
We don't have any such assumption about GOPATH pointing to same repository. PKGDIR is initialized as 
```PKGDIR="${GOPATH}/src/sigs.k8s.io/gcp-filestore-csi-driver"```

Reply:
So $PKGDIR/test/run-k8s-integration.sh may not be this script. Are we okay with that, or should we set PKGDIR to be the actual location of this script?

Reply:
As long as PGDIR is pointing to ${GOPATH}/src/sigs.k8s.io/gcp-filestore-csi-driver, the script is expected to be the present and same. Let's syncup offline and close on it

Reply:
Added a check to compare script dir path and the expected pkgdir